code-studio/src/js/levels/multi.js

```diff
@@ -220,6 +223,21 @@ Multi.prototype.getResult = function (dontAllowSubmit) {
   };
 };
 
+// called by external code that will display answer feedback
+Multi.prototype.getCurrentAnswerFeedback = function () {
+  if (!this.answersFeedback) {
+    return;
```

Comment:
I think `undefined` is quite reasonable. Calling code will treat them the same way. The lack of a return value is sort of accurate "I have no answer feedback". I find that I don't have to use `null` nearly as much in JavaScript as in C/C++